src/lib/rest/HttpHeaders.h

```diff
@@ -26,6 +26,37 @@
 * Author: Ken Zangelin
 */
 #include <string>
+#include <vector>
+
+#include "common/MimeType.h"
+
+
+
+// -----------------------------------------------------------------------------
```

Comment:
Wait, this was no mistake.

The text in the comment contains  `*/*` and thus would end the C-comment right there, resulting in a compilation error.
That's why I used C++ comments here.

NTC